lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -12,12 +12,12 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory, check_mongo_calls
 
-from ..signals import (
+from ..signals.handlers import (
```

Comment:
Since the code this tests is mostly in handlers.py now should we call this test_handlers?